include/utils.h

```diff
@@ -98,6 +100,10 @@ void get_ops(const smt::Term & term, smt::UnorderedOpSet & out);
  */
 bool is_lit(const Term & l, const Sort & boolsort);
 
+void cnf_to_dimacs(Term cnf, std::ostringstream & y);
+
+//Returns a string in DIMACs format for a given cnf formula
```

Comment:
```suggestion
//Returns a string in DIMACs format for a given cnf formula
void cnf_to_dimacs(Term cnf, std::ostringstream & y);
```
We usually document the function above its declaration.